src/models/todo.js

```diff
@@ -0,0 +1,12 @@
+export default class Todo {
```

Comment:
This class was perfectly done! It only does its job, creating a ToDo and returning it. Use this way of thinking for the Project class as well.